src/style-spec/expression/evaluation_context.js

```diff
@@ -9,7 +9,7 @@ import type { Feature } from './index';
 import type { Expression } from './expression';
 
 class EvaluationContext {
-    globals: {+zoom?: number};
+    globals: {zoom: number};
```

Comment:
Is the purpose here to help V8 optimize by providing a fixed object structure?  Note that for `heatmap-color`, we intend to pass `{heatmapDensity: number}`, rather than `{zoom: number}` as the globals -- but we could make it `{heatmapDensity: number, zoom: -1}` or something, since this optimization seems worth it.

Reply:
It's mainly to avoid an extra check in the implementation of `zoom` to satisfy flow.